programmers2/음양-더하기(수정).js

```diff
@@ -0,0 +1,8 @@
+function solution(absolutes, signs) {
+    var sum = 0;
+    for(let i = 0; i < absolutes.length; i++) {
+        if(signs[i]) sum += absolutes[i];
```

Comment:
공백 유의!
=== true를 수정하셨군요
굳입니당
```suggestion
    for (let i = 0; i < absolutes.length; i++) {
        if (signs[i]) sum += absolutes[i];
```